index.html

```diff
@@ -1093,7 +1079,6 @@ <h4 class="contact_head">Mubaris NK</h4>
 <script src="js/contestant.js"></script>
 <script src="js/projects.js"></script>
 <script src="js/issue-loader.js"></script>
-<script src="js/leaderBoard.js"></script>
```

Comment:
You actually deleted this new content present on site. Also there may be many such instances in code which are hard to figure out due to lot of changes.

Reply:
@pradeepgangwar I'll refork amd make a new PR... You may close this one.